src/Commands/Billable.php

```diff
@@ -26,6 +26,10 @@ class Billable extends Command {
   const MONTH = 'month';
   const FORTNIGHT = 'fortnight';
 
+  // @todo make this configurable.
+  const BILLABLE_PERCENTAGE = 0.8;
```

Comment:
ah I was just thinking you'd change the config file if you wanted to override it. Will look into this.